console/views/enterprise.py

```diff
@@ -164,6 +164,22 @@ class EnterpriseMyTeams(JWTAuthApiView):
     def get(self, request, enterprise_id, *args, **kwargs):
         name = request.GET.get("name", None)
         tenants = team_services.get_teams_region_by_user_id(enterprise_id, self.user, name)
+        tenant_names = {tenant["team_name"]: tenant for tenant in tenants}
+        usable_regions = region_repo.get_usable_regions(enterprise_id)
+        for usable_region in usable_regions:
+            region_tenants, total = team_services.get_tenant_list_by_region(enterprise_id, usable_region.region_id,
+                                                                            page=1,
+                                                                            page_size=9999)
+            for region_tenant in region_tenants:
+                tenant = tenant_names.get(region_tenant["tenant_name"])
+                if tenant:
+                    tenant["user_number"] = team_repo.get_tenant_users_by_tenant_ID(region_tenant["id"]).count()
+                    tenant["running_services"] = tenant.get("running_services", 0) + region_tenant["running_app_num"]
```

Comment:
What is required is the number of running applications, not the number of running components